<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInfoWidget</class>
 <widget class="QWidget" name="GameInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollGameInfo">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>724</width>
        <height>672</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelGameInfoTitle">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 217, 0);
font-weight: bold;
font-size: 18px;</string>
         </property>
         <property name="text">
          <string>Информация об игре</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetGameInfo" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="labelDateOfRelease">
            <property name="styleSheet">
             <string notr="true">color: rgbrgb(44, 188, 255);
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>22.11.1997</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDateOfReleaseInfo">
            <property name="styleSheet">
             <string notr="true">color: rgbrgb(44, 188, 255);
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Дата выхода</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" rowspan="3">
           <widget class="QTextBrowser" name="textGameDescription">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="autoFormatting">
             <set>QTextEdit::AutoAll</set>
            </property>
            <property name="documentTitle">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QListWidget" name="listTags">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelGameGenres">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 170, 127);
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Жанры</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelGameTags">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 170, 127);
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Тэги</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGameInfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(212, 255, 143);
font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Описание</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QListWidget" name="listGenres">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabGame">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabComments">
          <attribute name="title">
           <string>Комментарии</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </widget>
         <widget class="QWidget" name="tabGameNews">
          <attribute name="title">
           <string>Новости игры</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
